ipu-plugin/images/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM golang:1.20-alpine as builder
```

Comment:
Do we need a license header?

Reply:
I think we need  license header here too. I will update this.